chia/wallet/wallet_node.py

```diff
@@ -795,6 +795,8 @@ def is_new_state_update(cs: CoinState) -> bool:
 
         # We only process new state updates to avoid slow reprocessing. We set the sync height after adding
         # Things, so we don't have to reprocess these later. There can be many things in ph_update_res.
+        use_delta_sync = self.config.get("use_delta_sync", False)
+        min_height_for_subscriptions = fork_height if use_delta_sync else 0
```

Comment:
I don't believe you're *guaranteed* to have all the coin states up to this point already.

Reply:
Can you elaborate more on this?

Reply:
This is true, but I believe it's good enough for now until the new wallet protocol is released, as long as it's opt-in to enable this.